docs/reference/operations/lookup_transfers.md

```diff
@@ -10,3 +10,18 @@ An [`id`](../transfers.md#id) belonging to a [`Transfer`](../transfers.md).
 
 - If the transfer exists, return the [`Transfer`](../transfers.md).
 - If the transfer does not exist, return nothing.
+
+## Client libraries
+
+For language-specific docs see:
+
+* [Looking up transfers using the Java library](https://github.com/tigerbeetledb/tigerbeetle/tree/main/src/clients/java#creating-transfers)
+* [Looking up transfers using the Go library](https://github.com/tigerbeetledb/tigerbeetle/tree/main/src/clients/go#creating-transfers)
+* [Looking up transfers using the Node.js library](https://github.com/tigerbeetledb/tigerbeetle/tree/main/src/clients/node#creating-transfers)
```

Comment:
Is "creating transfers" the correct header to link to here?

Reply:
Good catch